### DISCUSSION TITLE
How to detect feedback_required ban on an any endpoint

### DISCUSSION THREAD

**Discussion by emreond** [sbertix/Swiftagram#177] [Q&A] [answered]
Helllo,

I am getting below error when trying to fetch User details with `Endpoint.User.summary(for: secret.identifier)` endpoint. In completion block, It goes to success case not failure. How can I detect this or any other ban error?

```{"message":"feedback_required","spam":true,"feedback_title":"Try Again Later","feedback_message":"We restrict certain activity to protect our community. Based on your use, this action will be unavailable for you until 2021-04-26. Tell us if you think we made a mistake.","feedback_url":"repute/report_problem/user_restriction_ENDPOINT_AUTOMATION_RESTRICT/?violation_type=22\u0026responsible_policy=user_restriction_Ig.Policy.BotDetection.SessionClassifier.BotDetectionGBDTtoTIESFriendshipFollowersPrediction","feedback_appeal_label":"Tell us","feedback_ignore_label":"OK","feedback_action":"report_problem","category":"user_restriction_ENDPOINT_AUTOMATION_RESTRICT","status":"fail"}```

**Comment by sbertix** [comment, marked as answer]
Hey @emreond, 

Every endpoint response conforming to `Specialized` has an `error` property populated with Instagram custom error messages. 
For the ones returning a `Wrapper` you can, as pointed out in your log, just return the message. 

```swift
let wrapper: Wrapper = /* assuming this is the wrapper */
if let status = wrapper.status.string(), status != "ok" {
   // Something went wrong…
   print(wrapper.message.string() ?? "Unknown error)
}
```

**Reply by emreond** [reply]
Aaaa ok thank you so much :)
